tests/test_integration_all_rust.py

```diff
@@ -690,6 +690,11 @@ def _ws_url(self):
     @max_logs(conn=4)
     def test_sentry_output(self):
         # Ensure bad data doesn't throw errors
+        ## The latest sentry library no longer uses the same
```

Comment:
I have a fix for this (capturing via the new sentry envelope API) here: https://github.com/mozilla-services/autopush-rs/compare/feat/1886-sentry...feat/1886-sentry-test